Khan/wonder-blocks
# [fix:dropdown]WB-637 - Fix testID on ActionMenu

## Summary
This PR fixes the `ActionMenu` by properly adding the `testId` prop to the opener. This will help us to unblock the deploy on Webapp.

## Test plan
1. Open https://deploy-preview-461--wonder-blocks.netlify.com/#!/ActionMenu/9
2. Make sure the `data-test-id` attribute is set to `teacher-menu`.

![dropdown-test-id](https://user-images.githubusercontent.com/843075/62231325-d2379a80-b391-11e9-810b-9992f8ea1967.png)
 on the `<button>` element.
